boards/samr21-xpro/include/periph_conf.h

```diff
@@ -240,6 +240,40 @@ static const spi_conf_t spi_config[] = {
 #define RTT_RUNSTDBY        (1)         /* Keep RTT running in sleep states */
 /** @} */
 
+/**
+ * @name ADC Configuration
+ * @{
+ */
+#define ADC_0_EN                           1
+#define ADC_MAX_CHANNELS                   14
+/* ADC 0 device configuration */
+#define ADC_0_DEV                          ADC
+#define ADC_0_IRQ                          ADC_IRQn
+/**
+ * @brief ADC Channel Configuration
+ */
+typedef struct {
```

Comment:
You are absolutely right. I will move it.
Regarding type name, I think I need to change the type name to `adc_conf_chan_t` to match other peripheral struct names?